### ISSUE TITLE
logging - linebreaks do not work, only first line is shown in db log

### ISSUE THREAD

**Issue by kaidaguerre** [turbot/steampipe#169]

**Comment by e-gineer** [comment]
Adding an example, partly to bump this issue... it makes troubleshooting difficult.

```
2021-04-29T21:32:37.523-0400 [WARN]  plugin.steampipe-plugin-algolia.plugin: [WARN]  listLog: result="{[{
2021-04-29T21:32:37.525-0400 [WARN]  plugin.steampipe-plugin-algolia.plugin: [WARN]  listLog: i="{
2021-04-29T21:32:37.525-0400 [WARN]  plugin.steampipe-plugin-algolia.plugin: [WARN]  listLog: i.Answer="
2021-04-29T21:32:37.525-0400 [WARN]  plugin.steampipe-plugin-algolia.plugin: [WARN]  listLog: i="{
2021-04-29T21:32:37.525-0400 [WARN]  plugin.steampipe-plugin-algolia.plugin: [WARN]  listLog: i.Answer="
2021-04-29T21:32:37.525-0400 [WARN]  plugin.steampipe-plugin-algolia.plugin: [WARN]  listLog: i="{
2021-04-29T21:32:37.525-0400 [WARN]  plugin.steampipe-plugin-algolia.plugin: [WARN]  listLog: i.Answer="
2021-04-29T21:32:37.525-0400 [WARN]  plugin.steampipe-plugin-algolia.plugin: [WARN]  listLog: i="{
2021-04-29T21:32:37.525-0400 [WARN]  plugin.steampipe-plugin-algolia.plugin: [WARN]  listLog: i.Answer="
2021-04-29T21:32:37.525-0400 [WARN]  plugin.steampipe-plugin-algolia.plugin: [WARN]  listLog: i="{
```

**Comment by binaek** [comment]
Dependant on https://github.com/turbot/steampipe-plugin-sdk/pull/606

**Comment by binaek** [comment]
Fixed by #3707
